content/.TEMPLATE.md

```diff
@@ -1,14 +1,15 @@
 ---
 layout: adr
-adr: 0
+adr: 0 # replace this number for the PR or ISSUE number
 title: Title of the ADR
 date: 2020-02-20
 status: Idea
```

Comment:
```suggestion
status: Idea, Draft, Review, Last Call, Final, Stagnant, Withdrawn, Living # pick one of these
```

Reply:
what about "Idea, Draft # pick one of these"?

Reply:
It makes sense. It's a template to start a ADR